<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>29</pointsize>
          </font>
         </property>
         <property name="text">
          <string>UWB系统监测</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>59</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>IP地址</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>20</y>
            <width>59</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>端口号</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Input_ip">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="Input_port">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>40</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="StartButton">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>20</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>20</y>
            <width>59</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>更新步长</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>20</y>
            <width>59</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>统计时间</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Input_time_step">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>40</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="Input_time">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>40</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>59</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>基站号</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>20</y>
            <width>59</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>门限值</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="input_gate_value">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>40</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="WriteGateButton">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>20</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>写入</string>
          </property>
         </widget>
         <widget class="QComboBox" name="station_list">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>104</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
